### PR TITLE
DT-74-feature-modal-message

### PR DESCRIPTION
**Pull request by Giang-st1-00**
@DuyTinh2003 @lamxuanthinh  review !

### CODE REVIEW

**Comment by lamxuanthinh** [comment]
oke

File: model/message.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export interface IMessageItems {
+    avatar: any;
```
Comment (Duy-Tinh-Dev) [inline model/message.ts:2]: !any

Comment (lamxuanthinh) [reply model/message.ts:2]: because this have still api yet @DuyTinh2003

**Comment by Duy-Tinh-Dev** [comment]
ok

**Pull request merged**